backend/src/main/java/com/htecgroup/skynest/controller/UserController.java

```diff
@@ -264,15 +269,18 @@ public ResponseEntity<String> confirmPasswordReset(
   }
 
   @Operation(summary = "Get User with that id")
+  @PreAuthorize("hasAuthority(T(com.htecgroup.skynest.model.entity.RoleEntity).ROLE_WORKER)")
   @GetMapping
   public List<UserResponse> getUsers() {
+    log.info("User {} accessed getusers", SecurityContextHolder.getContext().getAuthentication());
```

Comment:
Well if you are invoking SecurityContext just to print it, it is better not to do it. If you want tho, then Spring automatically can inject the Principal / Authentication as input param in the controller method.
My advice if you want to print which user triggers which actions for every log statement in the project, then you need to do it with Spring security filter and MD5 .

You know the drill, technical dept task in backlog

Reply:
I'll remove it.